---
layout: homepage
---

## About Me

I recieved my Bachelor degree from Sichuan Agricultrual Unviersity. Then I moved to Sichuan University and completed my master study in the lab of <a href="https://jingwanglab.org/" target="_black">**Jing Wang**</a> and <a href="https://scholar.google.com/citations?hl=zh-CN&user=9etBIG0AAAAJ&view_op=list_works&sortby=pubdate" target="_black">**Jianquan Liu**</a>. My master research primarily focused on the gene expression and regulation changes in the rediploidization of a recent Polyploid, Orychophragmus Violaceus. 

In the later year after my graduation of master, I was supported as a Research Assistant by Jing Wang to keep studying WGD and finish my master work.

Currently, I pursued my Ph.D journey at University of British Columbia (UBC) in September 2024, working in the lab of <a href="https://www.zoology.ubc.ca/mank-lab/" target="_black">**Judith Mank**</a>. My studies mainly focus on the sex chromosome evolution of guppy and its relatives.

## Research Interests
- **Sex Chromosomes Evolution**
- **Whole genome duplication** 
- **Gene duplication** 
- **Functional genomics** 

## Experience
- 2016.09 - 2020.06 : **BSc** in Life Science & Technology, Sichuan Agricultrural University
- 2020.09 - 2023.06 : **MCs** in Ecology, Sichuan University
- 2023.06 - 2024.06 : **Research Assistant** in Wang lab, Sichuan University
- 2024.09 - present : **PhD** in Zoology, University of British Columbia

## Publications

#2025

Wouter van der Bijl, Jacelyn J. Shu, Versara S. Goberdhan, Linley M. Sherin, **Changfu Jia**, María Cortázar-Chinarro, Alberto Corral-López, Judith E. Mank (2025) Deep learning reveals the complex genetic architecture of a highly polymorphic sexual trait. <a href="https://www.biorxiv.org/content/10.1101/2023.09.29.560175v3.abstract" target="_black">BioRxiv</a>


Qiang Lai, Zeng Wang, **Changfu Jia**, Xiner Qumu, Rui Wang, Zhipeng Zhao, Yao Liu, Yukang Hou, Jianquan Liu, Par K Ingvarsson, Jing Wang
(2025) Evolutionary responses and genomic consequences of polyploidization in natural populations of Orychophragmus. <a href="https://www.biorxiv.org/content/10.1101/2025.03.24.644964v2.abstract" target="_black">BioRxiv</a>

#2024

**Changfu Jia**, Qiang Lai, Yiman Zhu, Jiajun Feng, Xuming Dan, Yulin Zhang, Zhiqin Long, Jiali Wu, Zeng Wang, Xiner Qumu, Rui Wang, Jing Wang (2024) Intergrative metabolomic and transcriptomic analyses reveal the potential regulatory mechanism of unique dihydroxy fatty acid biosynthesis in the seeds of an industrial oilseed crop Orychophragmus violaceus. <a href="https://doi.org/10.1186/s12864-023-09906-0" target="_black">BMC genomics</a> 25: 29 **[IF=4.4]** 

Tingting Shi, Xinxin Zhang, Yukang Hou, **Changfu Jia**, Xuming Dan, Yulin Zhang, Yuanzhong Jiang, Qiang Lai, Jiajun Feng, Jianju Feng, Tao Ma, Jiali Wu, Shuyu Liu, Lei Zhang, Zhiqin Long, Liyang Chen, Nathaniel R Street, Pär K Ingvarsson, Jianquan Liu, Tongming Yin, Jing Wang (2024) The super-pangenome of Populus unveil genomic facets for its adaptation and diversification in widespread forest trees. <a href="https://doi.org/10.1016/j.molp.2024.03.009" target="_black">Molecular Plant</a> **[IF=27.5]** 

#2023

**Changfu Jia**, Yukang Hou, Qiang Lai, Yuling Zhang, Rui Wang, Jianquan Liu, Jing Wang (2023) A reference genome and its epigenetic landscape of potential Orychophragmus violaceus, an industrial crop species. <a href="https://www.biorxiv.org/content/10.1101/2023.09.21.558835v1.abstract" target="_black">BioRxiv</a>

Liyang Chen, Yuanzhong Jiang, Tingting Shi, **Changfu Jia**, Zhiqin Long, Xinxin Zhang, Yupeng Sang, Jianquan Liu, Jing Wang (2023) Chromosomal-level genome assembly of Populus davidiana <a href="https://doi.org/10.1101/2023.07.11.548481" target="_black">BioRxiv</a>

#2022

Yupeng Sang, Zhiqin Long, Xuming Dan, Jiajun Feng, Tingting Shi, **Changfu Jia**, Xinxin Zhang, Qiang Lai, Guanglei Yang, Hongying Zhang, Xiaoting Xu, Huanhuan Liu, Yuanzhong Jiang, Pär K Ingvarsson, Jianquan Liu, Kangshan Mao, Jing Wang (2022) Genomic insights into local adaptation and future climate-induced vulnerability of a keystone forest tree in East Asia. <a href="https://www.nature.com/articles/s41467-022-34206-8#citeas" target="_black">Nature communications</a> 13: 6541 **[IF=16.6]**

Shuyu Liu, Lei Zhang, Yupeng Sang, Qiang Lai, Xinxin Zhang, **Changfu Jia**, Zhiqin Long, Jiali Wu, Tao Ma, Kangshan Mao, Nathaniel R Street, Pär K Ingvarsson, Jianquan Liu, Jing Wang (2022) Demographic History and Natural Selection Shape Patterns of Deleterious Mutation Load and Barriers to Introgression across Populus Genome. <a href="https://doi.org/10.1093/molbev/msac008" target="_black">Molecular Biology and Evolution</a> 39(2): msac008 **[IF=10.7]**

Yukang Hou, Qi Lu, Jianxun Su, Xing Jin, **Changfu Jia**, Lizhe An, Yongke Tian, Yuan Song (2022) Genome-Wide Analysis of the HDAC Gene Family and Its Functional Characterization at Low Temperatures in Tartary Buckwheat (Fagopyrum tataricum). <a href="https://doi.org/10.3390/ijms23147622" target="_black">International Journal of Molecular Sciences</a> 39(2):2022; 23(14):7622 **[IF=0.7]**

